gns3/registry/config.py

```diff
@@ -236,7 +238,7 @@ def _add_qemu_config(self, new_config, appliance_config):
 
         new_config["kernel_command_line"] = appliance_config["qemu"].get("kernel_command_line", "")
 
-        new_config["qemu_path"] = "qemu-system-{}".format(appliance_config["qemu"]["arch"])
```

Comment:
The downside is we lost topology portability, because if we try to move a topology from an OS to another it will break.